abc.templates/infra/main.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  project_id                        = "REPLACE_PROJECT_ID"
+  automation_service_account_member = "REPLACE_AUTOMATION_SERVICE_ACCOUNT_MEMBER"
+}
+
+resource "google_project_service" "services" {
+  for_each = toset([
+    "iam.googleapis.com",
+    "iamcredentials.googleapis.com",
+    "sts.googleapis.com",
+  ])
+
+  project = local.project_id
+
+  service                    = each.value
+  disable_on_destroy         = false
+  disable_dependent_services = false
+}
+
+module "github_token_minter_github_wif" {
+  source = "git::https://github.com/abcxyz/github-token-minter.git//terraform/modules/github-wif?ref=c6c2a47034268b472dbf1e51c28118c5cedb61e4"
+
+  project_id = local.project_id
+
+  id = "token-minter"
+  github = {
+    owner_id   = "REPLACE_GITHUB_OWNER_ID"
+    owner_name = "REPLACE_GITHUB_OWNER_NAME"
+    repo_id    = "REPLACE_GITHUB_REPO_ID"
+    repo_name  = "REPLACE_GITHUB_REPO_NAME"
```

Comment:
@verbanicm  These repo values are only used within the module for setting `default_wif_attribute_condition`. However, because `wif_attribute_condition` is set, these values are not used, yet required. Should we make these optional?